docs/source/performance/caching.md

```diff
@@ -135,55 +144,47 @@ const resolvers = {
 }
 ```
 
-The `setCacheHint` method accepts an object with the same fields as [the `@cacheControl` directive](#in-your-schema-static).
+The `setCacheHint` method accepts an object with `maxAge` and `scope` fields.
 
-The `cacheControl` object also has a `cacheHint` field which returns the field's current hint. This object also has a few other helpful methods, such as `info.cacheControl.cacheHint.restrict({ maxAge, scope })` which is similar to `setCacheHint` but it will never make `maxAge` larger or change `scope` from `PRIVATE` to `PUBLIC`. There is also a function `info.cacheControl.cacheHintFromType()` which takes an object type from a GraphQL AST and returns a cache hint which can be passed to `setCacheHint` or `restrict`; it may be useful for implementing resolvers that return unions or interfaces.
+#### `cacheControl.cacheHint`
 
-### Root and composite-type fields are not cachable by default
```

Comment:
It looks like you've entirely removed this entire philosophy/motivation section, which I just wrote a few weeks ago. Having read discussion on GH issues over the years, I do think that having some sort of explanation for *why* cache control works the way it does rather than just *what* it is is helpful. I'm sure this section can be improved in many ways but I don't think it should be completely deleted.

Reply:
Okay great, I will restore and tweak!